packages/@aws-cdk/aws-appmesh/lib/http-route-path-match.ts

```diff
@@ -0,0 +1,247 @@
+import { CfnGatewayRoute, CfnRoute } from './appmesh.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * Path and Prefix properties for HTTP route match.
+ */
+export interface HttpRoutePathMatchConfig {
+  /**
+   * Route CFN configuration for path match.
+   *
+   * @default - no path match.
+   */
+  readonly wholePathMatch?: CfnRoute.HttpPathMatchProperty;
+
+  /**
+   * String defining the prefix match.
+   *
+   * @default - no prefix match
+   */
+  readonly prefixPathMatch?: string;
+}
+
+/**
+ * Defines HTTP route path or prefix request match.
+ */
+export abstract class HttpRoutePathMatch {
+  /**
+   * The value of the path must match the specified value exactly.
+   *
+   * @param path The exact path to match on
+   */
+  public static exact(path: string): HttpRoutePathMatch {
```

Comment:
Again, probably `exactly` is the name we want here.

Reply:
Addressed